specification/app/resource-manager/Microsoft.App/stable/2025-01-01/CommonDefinitions.json

```diff
@@ -294,6 +294,10 @@
           "x-ms-identifiers": [
             "triggerParameter"
           ]
+        },
+        "identity": {
+          "description": "The resource ID of a user-assigned managed identity that is assigned to the Container App, or 'system' for system-assigned identity.",
+          "type": "string"
```

Comment:
>   "type": "string" [](http://example.com/codeflow?start=7&length=19)

apply format:arm-id everywhere

Reply:
in our design, the value could be a constant string `system` or a resource id, so we can not add type contrain here